packages/react-components/src/components/FeedbackDialog/index.tsx

```diff
@@ -194,10 +224,35 @@ export const FeedbackDialog = ({
               />
 
               <StyledCardContent withAfterMessage={afterMessage !== undefined}>
-                <TextArea name="message" rows={4} autoFocus />
-                {errors && errors["message"] && (
-                  <Text color="red">{errors.message}</Text>
-                )}
+                <FormControl>
+                  {withEmailInput && (
+                    <Input
+                      name="email"
+                      type="email"
+                      placeholder="Your e-mail"
+                      autoFocus
+                    />
+                  )}
+                  {withEmailInput && (
+                    <Text color="comment">
+                      Providing an e-mail address is optional, but if you do so,
```

Comment:
the more i think about this, the more i lean to idea that additional fields should be rendered via a render prop which is provided by user and not hardcoded in the component.

Consider a case where we want to change this text. Changing it would
involve PR and release of `@questdb/react-components` as well as a
release of the consumer code.

Maybe something like this instead?

```
<FeedbackDialog
  onSubmit={(values) => {
    return Promise.resolve()
    // or, in case of failure
    return Promise.reject({ errors: [] })
  }}>
  {(Field) => (
    <Field label="Your email" info="Providing an e-mail address is optional...">
      <Input type="email" name="email" />
    </Field>
  )}
</FeedbackDialog>
```

wdyt?

Reply:
@argshook I thought about this and I like your idea, but there are two things I have issues with:

1. Fields are closely tied with API implementation in `saas-client`. My thinking is because of this, making the UI part dynamic will still involve changes & deployment of the Cloud UI, and therefore I'm not sure if it's worth future-proofing the code to that extent _with the current API implementation_. 
2. Render prop would mean that we need to make sure rendering is aligned with the rest of the `<FeedbackDialog`>` - in the current, simple solution, we know that it will be displayed the way is meant to be (i.e. no accidental positioning / width / padding issues).

Reply:
okay, let's roll with it for now and we'll see when or if we need to augment current code!